### ISSUE TITLE
Content from description fields almost invisible

### ISSUE THREAD

**Issue by StoyanDimitrov** [Lusito/web-ext-translator#22]
The "description" field accompanying every "message" is set as title if exists. When extension author has supplied a description field it should be clearly visible to the translators. It is meant to guide them and help decide on wording, provide context etc.

**Comment by Lusito** [comment]
I agree.. this was more of a temporary thing, as I hadn't had the time to figure out the best place for it. Maybe this is something for the sidebar (see #11) as well?

**Comment by StoyanDimitrov** [comment]
Could be in the sidebar but ensure good visibility. Do what is the most easier. Anything else will be better than a title.

I've prepared two of the layouts at the top of my head you can ignore (;

![wet](https://user-images.githubusercontent.com/10962709/44573386-0237f000-a776-11e8-84bc-3694d43512af.png)

Few notes.
- source and destination strings is a non editable list of the two selected languages side by side with a common scrollbar;
- selected source string text is, well, the whole text of the source string. When the string is huge scrollbar could be used;
- context is the group information and description;
- destination text editing textarea. Is the place where the translator does her job.

**Comment by rugk** [comment]
Exactly, put it into the sidebar. That's what I've porposed in https://github.com/Lusito/web-ext-translator/issues/11, too.
